com.gitlab.zehkira.Myuzi.yml

```diff
@@ -0,0 +1,21 @@
+app-id: com.gitlab.zehkira.Myuzi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: myuzi
+modules:
+  - python3-modules.json
+  - name: myuzi
+    buildsystem: simple
+    build-commands:
+      - make install-flatpak DESTDIR=/app
```

Comment:
You install scripts are breaking standards. The use of `DESTDIR` is only for staging, And as a result it will install things in `/app/usr` which is plain wrong. Instead it should honour prefix (that's what `/usr` is in your makefile)

To add to this, you shouldn't even need a flatpak specific target.

Reply:
I don't follow. Are you saying the command should be `make PREFIX=/app install-flatpak` and `DESTDIR` should be replaced with `PREFIX` in `install-flatpak` in the makefile?